api/controllers/ConfigController.js

```diff
@@ -25,7 +25,47 @@
  * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
  */
 
+/* globals Config, ConfigService */
+
 module.exports = {
 	
+  create: function(req, res) {
+
+    let key = req.body.data.attributes.key;
+    let value = req.body.data.attributes.value;
+
+    ConfigService.set(key, value)
+      .then(() => {
+        Config.find({
+          key : key
+        })
+        .then((createdEntry) => {
+          return res.created(createdEntry);
```

Comment:
Even if the action is named `created` to handle the incoming POST request, it must return a 200 HTTP code. Use `res.ok` instead of `res.created`.